Assets/Plugins/Linux/Core/LinuxPlatformSpecifics.cs

```diff
@@ -20,27 +20,16 @@
 * SOFTWARE.
 */
 
-#if UNITY_64 || UNITY_EDITOR_64
-#define PLATFORM_64BITS
-#elif UNITY_EDITOR_LINUX || UNITY_STANDALONE_LINUX
-#define PLATFORM_32BITS
-#endif
-
-#if UNITY_EDITOR
-#define EOS_DYNAMIC_BINDINGS
-#endif
+#if !EOS_DISABLE
 
 //#define ENABLE_CONFIGURE_STEAM_FROM_MANAGED
 using PlayEveryWare.EpicOnlineServices.Utility;
 using UnityEngine;
 using UnityEngine.Scripting;
 using System.Runtime.InteropServices;
 
-#if !UNITY_EDITOR_WIN && (UNITY_STANDALONE_LINUX || UNITY_EDITOR_LINUX)
-
-#if !UNITY_EDITOR_LINUX
-[assembly: AlwaysLinkAssembly]
-#endif
+// If standalone linux and not editor, or the linux editor
+#if (UNITY_STANDALONE_LINUX && !UNITY_EDITOR) || UNITY_EDITOR_LINUX
```

Comment:
For every platform _for which there is a Unity editor_ we need to have that platform's `PlatformSpecifics`-derived class be defined. That means the specifics class must be defined if either the editor for that platform is active, or if the target is the platform without any editor.